contrib/inventory/azure_rm.py

```diff
@@ -482,7 +491,7 @@ def network_client(self):
         if not self._network_client:
             self._network_client = self.get_mgmt_svc_client(NetworkManagementClient,
                                                             self._cloud_environment.endpoints.resource_manager,
-                                                            '2018-02-01')
+                                                            '2017-06-01')
```

Comment:
why this downgrade?

Reply:
not downgrade, but revert upgrading in previous commit